.gitignore

```diff
@@ -73,3 +73,18 @@ Local.xcconfig
 /buildServer.json
 
 *.orig
+
+# tuist
+**/Derived/
+
+# temporary workspace to test tuist
```

Comment:
Nit: should this comment be removed or modified before merging?

Reply:
💯 agree haha, removing it :)

Reply:
```suggestion
```